CanvasColorSpaceProposal.md

```diff
@@ -164,43 +159,67 @@ Note: An alternative approach that was considered was to augment the probablySup
 
 #### ImageBitmap
 
-TODO(ccameron): Review this
-ImageBitmap objects are augmented to have an internal color space attribute of type CanvasColorSpace and an internal pixelFormat attribute of type CanvasPixelFormat. The colorSpaceConversion creation attribute also accepts enum values that correspond to CanvasColorSpace values. Specifying a CanvasColorSpace value results in a conversion of the image to the specified color space.
+ImageBitmap objects (unless created with ``colorSpaceConversion="none"``) should keep track of their internal color space, and should store their contents at highest fidelity possible, subject to implementation limitations.
 
 #### ImageData
 
-TODO(ccameron): Review this
-
-IDL
+Add the following types to be used by `ImageData`.
 <pre>
-
-enum ImageDataStorageType {
+enum ImageDataEncoding {
```

Comment:
Noting that there was an accidental reference to the old "ImageDataStorageType" enum below, I wonder whether this change from "storage type" to "encoding" is warranted. There's precedent in the OpenGL API for the term "type" and in the Direct3D API for "format". Since JavaScript uses the term "Typed Arrays" I wonder whether this should just be left as "ImageDataStorageType" for symmetry.

Reply:
Updated all instances to ImageDataStorageType